docs/unified-test-documentation/dasharo-compatibility/330-docking-station-display-ports.md

```diff
@@ -0,0 +1,139 @@
+# Dasharo Compatibility: Docking station Display ports
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup#os-boot-from-disk).
+1. The docking station is connected to the USB-C port.
```

Comment:
```suggestion
1. The docking station is connected to the USB-C port using a USB-C to USB-C full-featured cable (>=5Gbps) with 5A current capability.
```

Display and fast data requires a full-featured USB-C to USB-C cable. Otherwise only USB 2.0 data will be available.

Power delivery at >60W requires the cable to have an e-marker that reports 5 amps current carrying capability. Otherwise the dock will refuse to deliver more than 60W (if it's a compliant USB-PD power source).

Reply:
The information has been added in a slightly different way to keep the documentation as universal as possible.